### PR TITLE
Standardized Readme

### PR DESCRIPTION
**Pull request by RichardLitt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,9 +7,15 @@
 
 > A collection of testing utilities for ipfs and libp2p
 
-## Documenation
+## Install
 
-See https://godoc.org/github.com/libp2p/go-testutil.
+```sh
+go get libp2p/go-testutil
```
Comment (whyrusleeping) [inline README.md]: this isnt right.

Comment (RichardLitt) [reply README.md]: How would I install this?

Comment (RichardLitt) [reply README.md]: Fixed.

**Review by whyrusleeping — APPROVED** [review]
LGTM

**Pull request merged**